### PR TITLE
bump cni plugins to 0.8.7

### PR DESCRIPTION
**Pull request by aojea**
we can keep using the official cni plugins releases

### CODE REVIEW

**Comment by aojea** [comment]
/assign @BenTheElder 
Thanks to @squeed :1st_place_medal:

**Comment by BenTheElder** [comment]
awesome. working on the images.

**Comment by BenTheElder** [comment]
this doesn't build, need to switch back to the upstream URL

**Comment by BenTheElder** [comment]
fixed it

**Comment by BenTheElder** [comment]
watching CI, then will bump the node image

**Comment by BenTheElder** [comment]
typical flakes, nothing alarming. pushing

File: images/base/Dockerfile [hunk 1]
```diff
@@ -101,7 +100,7 @@ RUN echo "Ensuring scripts are executable ..." \
     && rm -rf /tmp/crictl.tgz \
  && echo "Installing CNI binaries ..." \
     && export CNI_TARBALL="${CNI_VERSION}/cni-plugins-linux-${ARCH}-${CNI_VERSION}.tgz" \
-    && export CNI_URL="https://github.com/bentheelder/containernetworking-plugins/releases/download/${CNI_TARBALL}" \
```
Comment (aojea) [inline images/base/Dockerfile:104]: I don't know how I did build this locally :face_with_head_bandage: , but it created the container

**Pull request merged**